### DISCUSSION TITLE
Can't connect zkSync Sepolia testnet to Metamask

### DISCUSSION THREAD

**Discussion by Hajali-Selim** [Cyfrin/foundry-full-course-cu#2188] [Q&A] [answered]
Hello, I'm having trouble connecting the 'zkSync Sepolia testnet' to my Metamask wallet to test the bridging of the Blockchain Basics course.
I did get the 0.05 Sepolia ETH using a faucet, but when I go to Chainlist and click 'Connect Wallet', nothing happens. The website does not interact with my Metamask wallet. I tried connecting Metamask to the testnet manually using the info found [here](https://rpc.info/zksync-sepolia) but I aways get the following error.

![metamask_error](https://github.com/user-attachments/assets/781ec78c-4b3f-4969-96a3-0c274161a7d2)

I also tried bridging 0.01 Sepolia ETH into zkSync Sepolia testnet anyway, and they got lost.
Note: If I have my Exodus wallet installed, Chainlist will recognize it when I click 'Connect Wallet' but it is irrelevant here, since Exodus doesn't support zkSync Sepolia testnet.

Did anybody encounter such issues at this level? Thank you for your help.

**Comment by EngrPips** [comment, marked as answer]
Seem like you figured this out.
